manifest.json

```
@@ -1,28 +1,33 @@
 {
   "name": "__MSG_extensionName__",
   "short_name": "Humans.txt",
-  "author":"Joachim Robert",
-  "homepage_url":"https://github.com/joachimesque/humanstxt-webextension",
+  "author": "Joachim Robert",
```

At this point I think your name should also be listed as an author.

```suggestion
  "author": "Joachim Robert, Matteo Corossigo",
```

---

Thanks! Done in https://github.com/joachimesque/humanstxt-webextension/pull/4/commits/89761ab11829e86899641e121d9a680c4edd514c